build/simp-gpgkeys.spec

```diff
@@ -47,23 +47,23 @@ export PATH=/opt/puppetlabs/bin:$PATH
 
 # If we're a SIMP server, place the keys into the appropriate web directory
 
-dir='/var/www/yum/SIMP'
+dir='/var/www/yum/SIMP/GPGKEYS'
```

Comment:
I think both the old and new logic are wrong.  Per the comment, this was only supposed to copy the GPG keys, if this is a standard SIMP server installation. So if /var/www/yum/SIMP/GPGKEYS does not exist, IMHO the post install should do nothing..

Reply:
BTW  This kind of post install installation is frowned upon for RPMs.  RPM does not  keep track of these files, which means special logic needs to be created to remove them upon RPM uninstall.

Reply:
I updated it so it would only extract the files if the directory existed.  I am adding something to rake-helpers to put the keys in the repo when it is built from the ISO so maybe we won't need this in the future.